scipy/stats/tests/test_axis_nan_policy.py

```diff
@@ -720,3 +720,61 @@ def test_mixed_mask_nan_2():
              np.nan, ref1.pvalue, ref2.pvalue]
     np.testing.assert_array_equal(res.statistic, stat_ref)
     np.testing.assert_array_equal(res.pvalue, p_ref)
+
+
+def test_axis_None_vs_tuple():
+    # `axis` `None` should be equivalent to tuple with all axes
+    shape = (3, 8, 9, 10)
+    rng = np.random.default_rng(0)
+    x = rng.random(shape)
+    res = stats.kruskal(*x, axis=None)
+    res2 = stats.kruskal(*x, axis=(0, 1, 2))
+    np.testing.assert_array_equal(res, res2)
+
+
+@pytest.mark.parametrize(("axis"),
+                         list(permutations(range(-3, 3), 2)) + [(-4, 1)])
+def test_other_axis_tuples(axis):
+    # Check that _axis_nan_policy_factory treates all `axis` tuples as expected
+    rng = np.random.default_rng(0)
+    shape_x = (4, 5, 6)
+    shape_y = (1, 6)
+    x = rng.random(shape_x)
+    y = rng.random(shape_y)
+    axis_original = axis
+
+    # convert axis elements to positive
+    axis = tuple([(i if i >=0 else 3 + i) for i in axis])
```

Comment:
```suggestion
    axis = tuple([(i if i >= 0 else 3 + i) for i in axis])
```
`scipy/stats/tests/test_axis_nan_policy.py:747:29: E225 missing whitespace around operator`

Reply:
This change looks good, resolving.